pkg/deploy/controller/configchange/controller.go

```diff
@@ -52,6 +55,7 @@ func (c *DeploymentConfigChangeController) Handle(config *deployapi.DeploymentCo
 			if kerrors.IsConflict(err) {
 				return fatalError(fmt.Sprintf("DeploymentConfig %s updated since retrieval; aborting trigger: %v", deployutil.LabelForDeploymentConfig(config), err))
 			}
+			c.recorder.Eventf(config, "failedCreate", "Couldn't create initial deployment: %v", err)
```

Comment:
K

----- Original Message -----
> > @@ -52,6 +55,7 @@ func (c *DeploymentConfigChangeController) Handle(config
> > *deployapi.DeploymentCo
> >  			if kerrors.IsConflict(err) {
> >  				return fatalError(fmt.Sprintf("DeploymentConfig %s updated since
> >  				retrieval; aborting trigger: %v",
> >  				deployutil.LabelForDeploymentConfig(config), err))
> >  			}
> > +			c.recorder.Eventf(config, "failedCreate", "Couldn't create initial
> > deployment: %v", err)
> 
> Just to be 100% sure, I've verified this manually (letting my initial
> deployment rot, I got 2 events up front from the retry and another couple a
> few minutes later- both treated as dupes by k8s; the "count" increases for a
> single event).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3106/files#r32251088